test/e2e_node/gpu-device-plugin.go

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Maybe rename this file as gpu_device_plugin.go to be consistent with the other files in this directory?